uri-scheme.md

```diff
@@ -0,0 +1,35 @@
+# URI Scheme
+
+The `wormhole` URI Scheme is used to encode a wormhole code as a URI. This can then be used to generate QR codes, or be opened by the platform URI handler to open a supporting client.
+
+The general format looks like this, and assumes default values for all query fields:
+`wormhole:{code}`
```

Comment:
Since codes may more or less be arbitrary, maybe add a note on escaping here too